src/app/navigation/Home.screen.tsx

```diff
@@ -30,6 +31,11 @@ export const Home = () => {
     fetchMovies();
   }, []);
 
+  useEffect(() => {
```

Comment:
This useEffect is not necessary, you can use movies.length without a state and without a useEffect